R-package/R/utils.R

```diff
@@ -6,13 +6,8 @@ lgb.is.Dataset <- function(x) {
   lgb.check.r6.class(x, "lgb.Dataset") # Checking if it is of class lgb.Dataset or not
 }
 
-# use 64bit data to store address
-lgb.new.handle <- function() {
-  0.0 # Return numeric type in R
```

Comment:
@Laurae2 when using this function to allocate a zero pointer, it sometimes will get some strange values. This causes the incorrect result when calling lgb.is.null.handle .